### PR TITLE
feat: [SEI-3864]: updating the proxy routes.

### PR DESCRIPTION
**Pull request by harshilbits**
Updating the proxy routes and microfrontend types for SEI.
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by sharath-levelops — DISMISSED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

**Review by sharath-levelops — APPROVED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

**Comment by harshilbits** [comment]
trigger messagecheck

**Review by bhavya-sinha — APPROVED** [review]

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -206,6 +206,9 @@ module.exports = {
       ? process.env.SERVICE_DISCOVERY_URL || 'http://localhost:8888'
       : `${baseUrl}/servicediscovery`
   },
+  '/sei/api': {
+    target: targetLocalHost ? 'http://localhost:3000' : `${baseUrl}`
```
Comment (knagurski) [inline configs/devServerProxy.config.js:210]: Should there be a `/sei` after the `baseUrl` bit? You might also want to consider allow overriding the `localhost` with an environment variable.

Comment (harshilbits) [reply configs/devServerProxy.config.js:210]: `sei` is not required in the baseUrl , we have tested this. Sure will check for the usage of environment variable for localhost.

[the remark this answers is not in the document]
Comment (harshilbits) [reply configs/devServerProxy.config.js:210]: sure...will do in next PR.

**Pull request merged**